### ISSUE TITLE
Can't resolve 'fs' - React

### ISSUE THREAD

**Issue by aokdev06** [enesser/vCards-js#67]
Hi, what should I do to fix this error?? (usage for react.js)

`Error Message:`
```
ERROR in ./node_modules/vcards-js/index.js 37:17-30
Module not found: Error: Can't resolve 'fs' in '..\node_modules\vcards-js'
```

```
node   v16.15.0
npm    v8.13.2
yarn   v1.22.19
```

**Comment by aokdev06** [comment]
Problem solved with node-v14.17.6.

**Comment by zacBkh** [comment]
How to use this client side?
